cookbook/blending-with-context/README.md

```diff
@@ -0,0 +1,41 @@
+# Blending with Context
+
+There are situations when you need to blend the same token with multiple URIs but you want only one of them to be invoked depending on the context. For example a network support person should have access to an employee record but it needs to be scoped to only relative information with no edit rights while a system admin can have a full editable view of employee. Now there are two different roles trying to access the same entity but in different ways, this is where Context comes into play. 
+
+Contexts is an array of string that helps filter out the blending one step further. Two blending matches if both have the same token and context. In the previous example you can blend Employee with two different contexts:
+```
+Blender.MapUri<Employee>("/Employees/view/{?}", string[] { "View" });
+Blender.MapUri<Employee>("/Employees/edit/{?}", string[] { "Writable" });
+```
+All our prefab apps use Contexts with blending. For example some apps require a full view of Person from the [People](https://github.com/StarcounterApps/People) app while other only need only some basic info e.g. information that can be displyed in the search result. So we have two different views mapped to the same entity but with different contexts, the mapping can be achieved like:
```

Comment:
I'm not sure if the term "prefab apps" is good. Especially since no one really knows the distinction between a prefab app and a sample app. I think "sample app" would be easier to understand. But we don't have to deal with that now. 

while other only need only some basic info e.g. -> while others only need basic information, such as...

Reply:
Also, use "context" consistently, sometimes you capitalize, sometimes you don't.